.github/workflows/website_webhook.yml

```
@@ -1,67 +1,42 @@
-      sed -nr 's/.*"title": (.*), "updated.*/\1/p' <temp_title.txt >title.txt
-      echo $(cat title.txt)
-      if [[ $(cat title.txt) == *"Release"* ]]
+      category="${{ github.event.discussion.category.name }}"
+      title="${{ github.event.discussion.title }}"
+      body="${{ github.event.discussion.body }}"
+      
+      echo "::set-output name=title::${title}"
+      echo "::set-output name=summary::${body:0:500}"
+      echo "::set-output name=description::${body}"
+      
+      if [[ "${category}" == "Releases" ]]
       then
-        echo "::set-output name=release::true"
+        echo "::set-output name=is_release::true"
       else
-        echo "::set-output name=release::false"
-      fi
-
-  - name: Get summary
-    id: summary
-    env:
-      ISSUE_CONTEXT: ${{ toJson(github.event.issue) }}
-    run: |
-      echo $ISSUE_CONTEXT > temp.txt
-      sed -nr 's/.*### Brief description of the announcement(.*)### Detailed context.*/\1/p' <temp.txt >temp5.txt
-      echo "::set-output name=summary::$(cat temp5.txt)"
-      
-  - name: Get title
-    id: title
-    env:
-      ISSUE_CONTEXT: ${{ toJson(github.event.issue) }}
-    run: |
-      echo $ISSUE_CONTEXT > temp_title.txt
-      sed -nr 's/.*"title": (.*), "updated.*/\1/p' <temp_title.txt >title.txt
-      echo "::set-output name=title::$(cat title.txt)"
-
-  - name: Get description
-    id: description
-    env:
-      ISSUE_CONTEXT: ${{ toJson(github.event.issue) }}
-    run: |
-      echo $ISSUE_CONTEXT > temp.txt
-      sed -nr 's/.*"body": (.*), "closed_at.*/\1/p' <temp.txt >temp5.txt
-      echo "::set-output name=description::$(cat temp5.txt)"  
+        echo "::set-output name=is_release::false"
+      fi      
 
-  - name: Get current date
+  - name: Fetch current date
     id: date
     run: |
         echo "::set-output name=date::$(date +'%Y%m')"
         echo "::set-output name=date_spelled::$(date +'%B %Y')"
 
   - name: Repository Dispatch
-    uses: peter-evans/repository-dispatch@v1
+    uses: peter-evans/repository-dispatch@v1.1.3
     with:
       token: ${{ secrets.CODE_REPO_ACCESS_TOKEN }}
       repository: icub-tech-iit/code
       event-type: announcement_launched
-      client-payload: '{"description": ${{ steps.description.outputs.description }}, "id": "${{ steps.date.outputs.date }}", "title": ${{ steps.title.outputs.title }}, "date": "${{ steps.date.outputs.date_spelled }}", "summary": "${{ steps.summary.outputs.summary }}", "image": "???", "type" : "announcement_launched", "release": "${{ steps.rel_annou.outputs.release }}"}'
+      client-payload: '{"description": ${{ steps.announcement.outputs.description }}, "id": "${{ steps.date.outputs.date }}", "title": ${{ steps.announcement.outputs.title }}, "date": "${{ steps.date.outputs.date_spelled }}", "summary": "${{ steps.announcement.outputs.summary }}", "image": "???", "type" : "announcement_launched", "release": "${{ steps.announcement.outputs.is_release }}"}'
```

```suggestion
      client-payload: '{"description": "${{ steps.announcement.outputs.description }}", "id": "${{ steps.date.outputs.date }}", "title": "${{ steps.announcement.outputs.title }}", "date": "${{ steps.date.outputs.date_spelled }}", "image": "???", "type" : "announcement_launched", "category": "${{ steps.announcement.outputs.category }}"}'
```

Some changes here:
- We ran some tests and with the changes you did it requires the use of quotes on both the description and title (for the json format);
- We can remove the summary from the payload, since we would handle this from the website side;
- finally, we would replace the "release" payload with "category" instead.